mantle/cosa/schema_doc.go

```diff
@@ -638,7 +638,36 @@ var generatedSchemaJSON = `{
      "$id":"#/properties/gcp",
      "type":"object",
      "title":"GCP",
-     "$ref": "#/definitions/cloudartifact"
+     "required": [
+         "image",
+         "url",
+         "project"
```

Comment:
Can we make this optional for now? Otherwise interacting with older build metadata will fail. This will affect at least the upgrade tests, but possibly other places as well.

Reply:
👍 - in the interest of backwards compat (and not requiring a new schema version) we'll make project optional for now. I will note that image without project is not useful at all so it really is required, so we should make it required in the future. I would put this in a comment in the file but JSON isn't friendly to comments so I'll make it in an individual commit with a good commit message.

Reply:
pushed a new commit to address this concern.